owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -237,7 +238,7 @@ abstract class DrawerActivity : ToolbarActivity() {
     private fun setSpacesVisibilityBottomBar(uiResult: UIResult<OCCapability>) {
         if (uiResult is UIResult.Success) {
             val capabilities = uiResult.data
-            if (capabilities?.isSpacesAllowed() == true) {
+            if (AccountUtils.isFeatureSpacesAllowedForAccount(baseContext, account) && capabilities?.isSpacesAllowed() == true) {
```

Comment:
I see this condition is used several times along the code... Could we move it to a method in a common place?